docs/source/optimizers.mdx

```diff
@@ -178,13 +89,6 @@ class MyModule(torch.nn.Module):
 
 ```
 
-## API Docs[[optim_api_docs]]
-
-... under construction ...
-
-Here we'll provide further auto-generated API docs soon. Please feel free to contribute doc-strings for the respective optimizers, as `bitsandbytes` is a community effort.
-
-### StableEmbedding[[stable-emb-api]]
+## Next steps
 
-[[autodoc]] bitsandbytes.nn.StableEmbedding
-    - __init__
+For more conceptual details and explanation about 8-bit optimizers, take a look at the [8-bit optimizers](./explanations/optimizers) guide.
```

Comment:
Really like the new structure, much more graspable and gradual disclosure of details is dosed just right. Excellent!